code/common/range/index.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { ICONS } from "../../../assets/js/common/icons";
+
+const stars = [1, 2, 3, 4, 5];
```

Comment:
Use array func to create list and move length of it in const

Reply:
changed